includes/namespace.php

```diff
@@ -80,6 +80,35 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) ||
+	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
+) {
+	add_action(
+		'admin_notices',
+		static function () {
+			?>
+			<div class="notice notice-error">
+				<p>
+					<strong><?php esc_html_e( 'Web Stories plugin could not be initialized.', 'web-stories' ); ?></strong>
+				</p>
+				<p>
+					<?php
+					printf(
+						/* translators: %s: build commands. */
+						__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ), // phpcs:ignore WordPress.Security.EscapeOutput
+						'<code>composer install &amp;&amp; npm install &amp;&amp; npm run build</code>'
+					)
+					?>
+				</p>
+			</div>
+			<?php
+		}
+	);
+
+	return;
```

Comment:
Need to skip this for tests, because `assets/js/edit-story.js` does not exist there.

Reply:
Can't we just build the js for the tests?

Reply:
We can, but I don't want to make building JS a dependency for running tests if we can avoid it.

Reply:
I normally build js, as it builds the asset files as well which the php should really have them